src/System.Text.Encoding.Extensions/tests/4.0.10.0/Encoding/project.json

```diff
@@ -0,0 +1,16 @@
+{
+  "dependencies": {
+    "System.Globalization": "4.0.10-beta-22809",
+    "System.Collections": "4.0.10-beta-22809",
+    "System.Console": "4.0.0-beta-22809",
+    "System.Runtime": "4.0.20-beta-22809",
+    "System.Runtime.Extensions": "4.0.10-beta-22809",
+    "System.Text.Encoding": "4.0.10-beta-22809",
+    "System.Text.Encoding.Extensions": "4.0.10-beta-22809",
```

Comment:
target project should be picked up by project 2 project reference, not from nuget
e.g. see #1138